code/game/gamemodes/changeling/changeling.dm

```diff
@@ -131,10 +131,16 @@ GLOBAL_LIST_INIT(slot2type, list("head" = /obj/item/clothing/head/changeling, "w
 	changeling.objectives += absorb_objective
 
 	if(prob(60))
-		var/datum/objective/steal/steal_objective = new
-		steal_objective.owner = changeling
-		steal_objective.find_target()
-		changeling.objectives += steal_objective
+		if(prob(70))
+			var/datum/objective/steal/steal_objective = new
+			steal_objective.owner = changeling
+			steal_objective.find_target()
+			changeling.objectives += steal_objective
+		else

```

Comment:
Add a check to prevent players from receiving multiple download objectives.

Reply:
this proc assigns all objectives at once it doesn't matter.

Reply:
oops I clicked on the wrong code. I meant in traitor `forge_single_objective`

Reply:
fixed